### PR TITLE
Add a config item for compliance checker integration

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Add `compliance_checker_enabled` in preparation for the rollout next week (disabled by default).

### CODE REVIEW

File: cluster/node-pools/master-default/userdata.clc.yaml [hunk 1]
```diff
@@ -501,7 +501,7 @@ storage:
                 value: https://identity.zalando.com/.well-known/openid-configuration
               - name: ENABLE_INTROSPECTION
                 value: "true"
-          - image: registry.opensource.zalan.do/teapot/image-policy-webhook:v0.4.1
+          - image: registry.opensource.zalan.do/teapot/image-policy-webhook:{{{if eq .Cluster.ConfigItems.compliance_checker_enabled "true"}}}master-44{{{else}}}v0.4.1{{{end}}}
```
Comment (mikkeloscar) [inline cluster/node-pools/master-default/userdata.clc.yaml]: Too many `{` here, only two is needed :)

Comment (aermakov-zalando) [reply cluster/node-pools/master-default/userdata.clc.yaml]: Oops!

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**